Description: remove reference to firebird.init.d.suse.in
 That file isnot present in the repackaged source, since it cannot
 be distributed by Debian -- has copyright statement without license.
Author: Damyan Ivanov <dmn@debian.org>
Forwarded: no

--- a/configure.ac
+++ b/configure.ac
@@ -1244,7 +1244,6 @@ case "$PLATFORM" in
         gen/install/misc/firebird.xinetd:builds/install/arch-specific/linux/firebird.xinetd.in
         gen/install/misc/firebird.init.d.generic:builds/install/arch-specific/linux/firebird.init.d.generic.in
         gen/install/misc/firebird.init.d.mandrake:builds/install/arch-specific/linux/firebird.init.d.mandrake.in
-        gen/install/misc/firebird.init.d.suse:builds/install/arch-specific/linux/firebird.init.d.suse.in
         gen/install/misc/firebird.init.d.debian:builds/install/arch-specific/linux/firebird.init.d.debian.in
         gen/install/misc/firebird.init.d.gentoo:builds/install/arch-specific/linux/firebird.init.d.gentoo.in
         gen/install/misc/firebird.init.d.slackware:builds/install/arch-specific/linux/firebird.init.d.slackware.in
